cmd/lakefs/cmd/migrate.go

```diff
@@ -162,8 +141,37 @@ var upCmd = &cobra.Command{
 		}
 		defer kvStore.Close()
 
-		_ = mustValidateSchemaVersion(ctx, kvStore)
-		fmt.Printf("No migrations to apply.\n")
+		force, _ := cmd.Flags().GetBool("force")
+
+		// -- This part contains the migration to ACL and will be removed in next version
+
+		// skip migrate to ACL for users with External authorizations
+		if !cfg.IsAuthTypeAPI() {
+			now := time.Now()
+			var doUpdate bool
+			if force {
+				doUpdate = true
+			} else {
+				doUpdate = migrateToACL(ctx, kvStore, cfg, logging.Default(), false, now)
+			}
+			if !doUpdate {
+				fmt.Println("Updated nothing.  Re-run with --force to update!")
+				return
+			}
+
+			migrateToACL(ctx, kvStore, cfg, logging.Default(), true, now)
+		}
+
+		err = kv.SetDBSchemaVersion(ctx, kvStore, kv.ACLMigrateVersion)
+		if err != nil {
+			// this should not happen
+			fmt.Println("migration succeeded - failed to upgrade version")
```

Comment:
What's the action item here? Should the user rerun?

Reply:
Added thanks to printed message.
IMO A better way would be to run `migrate goto --force`, but I didn't want to insert this decision as part of this PR